src/Sulu/Bundle/ContactBundle/Tests/Functional/Infrastructure/Sulu/Search/AccountReindexProviderTest.php

```diff
@@ -0,0 +1,149 @@
+        $this->assertNotNull($results[0]['createdAt']);
+
+        $this->assertSame(AccountInterface::RESOURCE_KEY . '::' . $account2->getId(), $results[1]['id']);
+        $this->assertSame('Test Account 2', $results[1]['title']);
+    }
+
+    public function testProvideWithSpecificIdentifiers(): void
+    {
+        $account1 = $this->createAccount('Account One');
+        $account2 = $this->createAccount('Account Two');
+        $account3 = $this->createAccount('Account Three');
+
+        $this->entityManager->flush();
+
+        $identifiers = [
+            AccountInterface::RESOURCE_KEY . '::' . $account1->getId(),
+            AccountInterface::RESOURCE_KEY . '::' . $account3->getId(),
+        ];
+
+        $config = ReindexConfig::create()
+            ->withIndex('admin')
+            ->withIdentifiers($identifiers);
+
+        $results = \iterator_to_array($this->provider->provide($config));
+
+        $this->assertCount(2, $results);
+
+        $resultTitles = \array_column($results, 'title');
+        $this->assertContains('Account One', $resultTitles);
+        $this->assertContains('Account Three', $resultTitles);
+        $this->assertNotContains('Account Two', $resultTitles);
+    }
+
+    public function testProvideWithChangedAndCreatedDates(): void
+    {
+        $account = $this->createAccount('Test Account');
+
+        $changedDateString = '2023-06-01 15:30:00';
+        $createdDate = new \DateTimeImmutable('2023-01-01 10:00:00');
+        $changedDate = new \DateTimeImmutable($changedDateString);
+
+        $account->setCreated($createdDate);
+
+        $this->entityManager->flush();
+
+        $connection = self::getEntityManager()->getConnection();
+        $sql = 'UPDATE co_accounts SET changed = :changed WHERE id = :id';
+
+        $connection->executeStatement($sql, [
+            'changed' => $changedDateString,
+            'id' => $account->getId(),
+        ]);
+
+        $config = ReindexConfig::create()->withIndex('admin');
+        $results = \iterator_to_array($this->provider->provide($config));
+
+        $this->assertCount(1, $results);
+        $result = $results[0];
+
+        $this->assertSame($createdDate->format('c'), $result['createdAt']);
+        $this->assertSame($changedDate->format('c'), $result['changedAt']);
+    }
+
+    public function testProvideWithEmptyIdentifiers(): void
+    {
+        $this->createAccount('Test Account');
+        $this->entityManager->flush();
+
+        $config = ReindexConfig::create()
+            ->withIndex('admin')
+            ->withIdentifiers([]);
+
+        $results = \iterator_to_array($this->provider->provide($config));
+
+        $this->assertCount(1, $results);
+    }
+
+    private function createAccount(string $name): Account
+    {
+        $account = new Account();
+        $account->setName($name);
+        $account->setCreated(new \DateTimeImmutable());
+        $account->setCreated(new \DateTimeImmutable());
```

Comment:
The `setCreated` called twice should be `setChanged`.

Use two different dates which you match in the first test:

```suggestion
        $account->setCreated(new \DateTimeImmutable('2000-01-01 12:00:00'));
        $account->setChanged(new \DateTimeImmutable('2024-12-24 23:00:00'));
```